stdlib/format.mli

```diff
@@ -16,14 +16,37 @@
 (** Pretty-printing.
 
    This module implements a pretty-printing facility to format values
-   within 'pretty-printing boxes'. The pretty-printer splits lines
-   at specified break hints, and indents lines according to the box
-   structure.
+   within {{!boxes}'pretty-printing boxes'} and {{!tags}'semantic tags'}
+   combined with a set of {{!fpp}printf-like functions}.
+   The pretty-printer splits lines at specified {{!breaks}break hints},
+   and indents lines according to the box structure.
+   Similarly, {{!tags}semantic tags} can be used to decouple text
+   presentation from its contents.
 
+   This printer-printing facility is implemented as an overlay on top of
+   abstract {{!section:formatter}formatters} which provide basic output
+   functions.
+   There are three predefined formatters:
+   - {!std_formatter} outputs to {{!Pervasives.stdout}stdout}
+   - {!err_formatter} outputs to {{!Pervasives.stderr}stderr}
+   - {!str_formatter} outputs to the {!stdbuf} {{!Buffer}buffer}
```

Comment:
Good point, I agree that `str_formatter` should not be here. I would rather keep for now the `std,err_formatter` part since it is needed in the current structure of the documentation to explain why all functions are duplicated.